packages/instantsearch.css/src/themes/reset.scss

```diff
@@ -196,3 +196,61 @@
   text-align: center;
   transform: translateX(-50%);
 }
+
+// Reset Carousel
+
+.ais-Carousel {
+  position: relative;
+}
+
+.ais-Carousel-list {
+  grid-auto-columns: 42%;
+  display: grid;
+  grid-auto-flow: column;
+  overflow-x: auto;
+  scroll-behavior: smooth;
+  scroll-snap-type: x proximity;
+}
+
+@media (min-width: 999px) {
+  .ais-Carousel-list {
+    grid-auto-columns: 20%;
+  }
+}
+
+.ais-Carousel-navigation {
+  position: absolute;
+  top: 35%;
+  z-index: 1;
+}
+
+@media (hover: none) and (pointer: coarse) {
+  .ais-Carousel-navigation {
+    display: none;
+  }
+}
+
+.ais-Carousel:hover .ais-Carousel-navigation:not([hidden]) {
+  opacity: 1;
+  visibility: visible;
+}
```

Comment:
this should be part of the specific styles IMO

Reply:
You mean buttons should always be visible in `reset` ?

Reply:
yes, I think those styles would need to be reverted in many cases IMO